themes/my-test-theme-dark/color.yaml

```diff
@@ -0,0 +1,25 @@
+# This is a sample theme used for demo
```

Comment:
Oh also could we add the YAML language server comment here too?

```suggestion
# yaml-language-server: $schema=https://github.com/kubecolor/kubecolor/raw/main/config-schema.json

# This is a sample theme used for demo
```

Reply:
It's done, but there's an issue with the `data.theme.durationFresh` that is `durationFresh` or `durationfresh`... It's just json validation for now so we'll sort out later. I need to merge this as I'll be unavailable soon.

Reply:
Well when viper reads yaml it does so in a case insensitive way. So it shouldn't matter much